crates/node/src/cli.rs

```diff
@@ -44,12 +44,7 @@ pub struct Config {
     )]
     pub node_key_file: PathBuf,
 
-    #[arg(
-        long,
-        long_help = "",
-        env = "GEVULOT_P2P_DISCOVERY_ADDR",
-        default_value = "34.88.251.176:9999"
-    )]
+    #[arg(long, long_help = "", env = "GEVULOT_P2P_DISCOVERY_ADDR")]
```

Comment:
Why drop the default value?

Reply:
To manage the case the node start without connecting, sort of primary bootstraper. In this case, you specify nothing and the node didn't try to connect.